mappings/net/minecraft/world/loot/function/LootTableFunctionEnchantRandomly.mapping

```diff
@@ -1,4 +1,21 @@
 CLASS none/bbo net/minecraft/world/loot/function/LootTableFunctionEnchantRandomly
+	CLASS none/bbo$a Serializer
+		METHOD a deserialize (Lcom/google/gson/JsonObject;Lcom/google/gson/JsonDeserializationContext;[Lnone/bcc;)Lnone/bbo;
+			ARG 0 json
+			ARG 1 context
+			ARG 2 conditions
+		METHOD a serialze (Lcom/google/gson/JsonObject;Lnone/bbo;Lcom/google/gson/JsonSerializationContext;)V
+			ARG 0 json
+			ARG 1 functionClass
+			ARG 2 context
+		METHOD a serialze (Lcom/google/gson/JsonObject;Lnone/bbq;Lcom/google/gson/JsonSerializationContext;)V
```

Comment:
`serialize` - other classes seem to have the same typo.
Also are there two identical de/serialize methods?